Server/api/routes/referee/matchups.ts

```diff
@@ -91,6 +91,43 @@ refereeMatchupsRouter.$get<{ matchup: MatchupInterface }>("/:tournamentID/:match
     };
 });
 
+// TODO: Move this to a general matchup endpoint later
```

Comment:
how "not general" is this? i must be missing something :P

Reply:
iw as basically thinking that whenever it gets to the point where the match page can finally be made, then it could be an endpoint that's moved to https://github.com/Corsace/corsace/blob/master/Server/api/routes/matchup.ts which would be used for that match info page as well alongside ref pages to see chat log history